src/support/utf8.c

```diff
@@ -615,71 +615,63 @@ size_t u8_printf(const char *fmt, ...)
     return cnt;
 }
 
-/* based on the valid_utf8 routine from the PCRE library by Philip Hazel
+/* Rewritten completely, original code not based on anything else
 
    length is in bytes, since without knowing whether the string is valid
    it's hard to know how many characters there are! */
-int u8_isvalid(const char *str, size_t length)
+int u8_isvalid(const char *iStr, size_t iLength)
```

Comment:
Is using that convention a problem?  (I dislike Hungarian myself, this was something that the other developers had wanted, just for the parameters, similar to Google's requirement of putting a "p" on formal parameter names...).  I hadn't found any C/C++ style guide for Julia internals, just the guide for Julia code.  Thanks!